.spelling

```diff
@@ -68,3 +68,23 @@ Pterothorax
 resized
 encodings
 autofill
+overlines
+strikethroughs
+bitmapped
+OpenGL
+0x00
```

Comment:
I would have preferred to put these in code like `0x00` and then the spell checker should ignore them.